okhttp/src/main/java/com/squareup/okhttp/internal/spdy/SpdyConnection.java

```diff
@@ -147,14 +147,14 @@ private SpdyConnection(Builder builder) throws IOException {
       okHttpSettings.set(Settings.INITIAL_WINDOW_SIZE, 0, OKHTTP_CLIENT_WINDOW_SIZE);
     }
 
-    hostName = builder.hostName;
+    service = protocol + ":" + builder.hostName + ":" + builder.socket.getPort();
```

Comment:
Oh, in alternate services, it is ( alpn, host, port )

Maybe I shouldn't tie to alt svc, but the bit I was troubleshooting when
looking at this had to do w/ protocol nego.

Other ideas?